components/brave_rewards/browser/extension_rewards_service_observer.cc

```diff
@@ -264,4 +264,23 @@ void ExtensionRewardsServiceObserver::OnPendingContributionSaved(
   event_router->BroadcastEvent(std::move(event));
 }
 
+void ExtensionRewardsServiceObserver::OnRemovePendingContribution(
```

Comment:
where is this being used? Are pending contributions displayed in both webui and the extension?

Reply:
yes